### PR TITLE
Docs: Issue 1636: modularize Monitoring chapter

### PR DESCRIPTION
**Pull request by jenmalloy**

### CODE REVIEW

File: documentation/modules/proc-deploy-grafana.adoc [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+----
+
+. Expose the Grafana service:
++
+[options="nowrap",subs="attributes"]
+----
+{cmdcli} expose service grafana
+----
+
+Grafana accepts the username `admin` and password `admin` by default. For more information about how to connect Grafana to Prometheus, see the link:https://prometheus.io/docs/visualization/grafana/#creating-a-prometheus-data-source[Prometheus documentation]. Use `prometheus.enmasse.svc.cluster.local` as the Prometheus host name.
```
Comment (ncbaratta) [inline documentation/modules/proc-deploy-grafana.adoc]: External link gets a ^

File: documentation/modules/proc-deploy-grafana.adoc [hunk 2]
```diff
@@ -0,0 +1,21 @@
+[id='deploy-grafana-{context}']
+= Deploying Grafana
+
+.Procedure
+
+. Create the Grafana deployment:
```
Comment (ncbaratta) [inline documentation/modules/proc-deploy-grafana.adoc]: Shouldn't this just be 'deploy grafana'?

File: documentation/modules/proc-deploy-grafana.adoc [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
++
+[options="nowrap",subs="attributes"]
+----
+{cmdcli} create -f ./resources/grafana/grafana.yaml -n enmasse
```
Comment (ncbaratta) [inline documentation/modules/proc-deploy-grafana.adoc]: is it always gonna be enmasse?

File: documentation/modules/proc-deploy-grafana.adoc (hunk 1 above)
Comment (ncbaratta) [inline documentation/modules/proc-deploy-grafana.adoc]: I feel like the 'use...' sentence at the end doesn't really belong here. Is this a Tip or a Note?

File: documentation/modules/proc-deploy-prometheus.adoc [hunk 4]
```diff
@@ -0,0 +1,24 @@
+[id='deploy-prometheus-{context}']
+= Deploying Prometheus
+
+.Prerequisites
+* For Prometheus to monitor pods in the cluster, cluster-admin privileges are required.
```
Comment (ncbaratta) [inline documentation/modules/proc-deploy-prometheus.adoc]: is cluster-admin a literal?

File: documentation/modules/proc-deploy-prometheus.adoc [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+.Procedure
+
+. Create the Prometheus deployment:
```
Comment (ncbaratta) [inline documentation/modules/proc-deploy-prometheus.adoc]: 'Deploy Prometheus'

File: documentation/modules/proc-deploy-prometheus.adoc [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
++
+[options="nowrap",subs="attributes"]
+----
+{cmdcli} create -f ./resources/prometheus/prometheus.yaml -n enmasse
+----
+
+ifeval::["{cmdcli}" == "oc"]
+. Grant cluster-reader privileges to the Prometheus service account:
```
Comment (ncbaratta) [inline documentation/modules/proc-deploy-prometheus.adoc]: is 'cluster-reader' a literal?

File: documentation/modules/proc-deploy-grafana.adoc (hunk 3 above)
Comment (jenmalloy) [reply documentation/modules/proc-deploy-grafana.adoc]: @lulf: is `enmasse` a variable here?

**Comment by ncbaratta** [comment]
[Peer review on PR 1637 for Jen](https://trello.com/c/SOb3wgQY/636-peer-review-on-pr-1637-for-jen)

**Pull request merged**